modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -28,13 +28,17 @@ The subnets assigned to nodes and the IP addresses assigned to individual pods a
 
 While the OVN-Kubernetes network provider implements many of the capabilities present in the OpenShift SDN network provider, the configuration is not the same.
 
-If your cluster uses any of the following OpenShift SDN capabilities, you must manually configure the same capability in OVN-Kubernetes:
-
+* If your cluster uses any of the following OpenShift SDN capabilities, you must manually configure the same capability in OVN-Kubernetes:
++
+--
 * Namespace isolation
 * Egress IP addresses
 * Egress network policies
 * Egress router pods
 * Multicast
+--
+
+* If your cluster uses the any part of the `100.64.0.0/16` IP address range, you cannot migrate to OVN-Kubernetes because it uses this IP address range internally.
```

Comment:
we should also include the v6 join subnet `fd98::/64` here.

Reply:
nit: If your cluster uses ~the~ any part of the...

Reply:
@abhat, but isn't that only for dual-stack or IPv6 stack clusters?

Reply:
Theoretically, we always reserve the v6 join subnet irrespective of whether the cluster is dual-stack. But the subnet conflict becomes germane only for dual-stack or v6 clusters. My point was for documentation completeness, we should also document v6 join subnets in case of v6 clusters.

Reply:
We don't support migration to IPv6 OVNKubernetes so far, as 1) OpenShiftSDN doesn't support IPv6, 2) the current migration approach requires that the service IPs keep unchanged.

Reply:
@pliurh that's a very good point. If we won't ever have conflicts on v6 join subnets because openshift-sdn doesn't support v6, we can remove that. @jboxman, sorry for the noise.